StackTracerCore/Debugger.cs

```diff
@@ -46,175 +47,168 @@ private static void TraceWarning(string message, Exception ex)
         public static List<ManagedThread> CollectTraces(int processId, string outputDirectory)
         {
             Logger.Init(string.Empty, outputDirectory, "StackTracerCore", true);
-            List<ManagedThread> stacks = new List<ManagedThread>();
+            var stacks = new List<ManagedThread>();
             StackTracerStats stats = new StackTracerStats
             {
                 StatsType = "StackTracer",
                 SiteName = SiteName
             };
 
-            try
-            {
-                DateTime startTime = DateTime.UtcNow;
-                TraceLine("CollectTraces Function started");
+            Stopwatch sw = new Stopwatch();
+            sw.Start();
 
-                DataTarget target = null;
-                DateTime processAttachedTime = DateTime.MinValue;
+            TraceLine("CollectTraces Function started");
+            DataTarget dt = CreateDataTarget(processId);
+            sw.Stop();
```

Comment:
You don't have to call Stop here.  Just using `sw.ElapsedMilliseconds` will be good enough.

Reply:
Same for below.

It's ok to call Stop here, but in practice it's not really going to do much versus omitting the call in this code.

Reply:
done!